RabbitMq/BatchConsumer.php

```diff
@@ -98,8 +106,10 @@ public function setCallback($callback)
         return $this;
     }
 
-    public function consume()
+    public function consume(int $batchAmount = 0)
     {
+        $this->target = $batchAmount;
```

Comment:
What is the purpose of target? And why does it get the value of batchAmount? Shouldn't batchAmount then be called target?

Reply:
You are right, I could find a better name for $batchAmount. Thank you.

Reply:
Renamed it in: https://github.com/eMAGTechLabs/RabbitMqBundle/pull/42/files#diff-6a0368091c180580d0ab8b72e4c7edcd7d93e247ad1b75e212d4630f319fa844